ad9361/sw/ad9361_api.c

```diff
@@ -601,7 +601,7 @@ int32_t ad9361_get_en_state_machine_mode (struct ad9361_rf_phy *phy,
 int32_t ad9361_set_rx_rf_gain (struct ad9361_rf_phy *phy,
 							   uint8_t ch, int32_t gain_db)
 {
-	struct rf_rx_gain rx_gain = {0};
+	struct rf_rx_gain rx_gain = {0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
hmm ; which compiler gives this error ?
typically, `rx_gain = {0};` & `rx_gain = {0, 0, 0, 0, 0, 0, 0, 0, 0};` should be equivalent ; I do know that for GCC this is true

Reply:
Clang compiler for linux build : https://travis-ci.org/amiclaus/no-OS/jobs/429935069#L664

Reply:
thanks for the info ;
i suspected this behavior may be only GCC specific ;